src/processors/index.ts

```diff
@@ -92,12 +109,23 @@ export class Processors {
           modes.push(Mode.FULL);
           break;
         default:
-          throw new Error("No avaiable mode from mainnet smart contract API");
+          throw new Error("No avaiable mode from PARENTNET smart contract API");
+      }
+    }
+    if (config.reverseRelayerCsc.isEnabled){
+      const subnetSmartContractMode = await this.subnetService.Mode(); 
+      switch (subnetSmartContractMode) {
+        case "reverse full":
```

Comment:
let's use `reverse_full`. don't leave env/config with space in it

Reply:
fixed